### PR TITLE
feat: switch from Windows to Linux on-prem deployments

### PR DESCRIPTION
**Pull request by paulswartz**
#### Summary of changes
**Asana Ticket:** [get Trike-dev running in the Linux-staging cluster](https://app.asana.com/0/0/1204806195243075/f)

Note from @nlwstein - I have updated this PR to also include a workflow to target our new production instance, and removed the existing Windows production workflows to support these new environments. 

#### Reviewer Checklist
- [ ] Any new or changed functions have typespecs
- [ ] Tests were added for any new functionality (don't just rely on Lcov)
- [ ] Meets ticket's acceptance criteria

### CODE REVIEW

**Comment by paulswartz** [comment]
@bklebe the first production Linux instance is online if you wanted to take this up while I'm OOO.

**Comment by bklebe** [comment]
@nlwstein

File: .github/workflows/dev.yml [hunk 1]
```diff
@@ -1,41 +1,40 @@
-name: Deploy to Dev
+name: Deploy to Dev (ECS)
 
 on:
   workflow_dispatch:
   push:
-    branches: [main]
+    branches: [main, ps-linux]
```
Comment (Whoops) [inline .github/workflows/dev.yml:6]: The `ps-linux` branch should probably be removed before merging.

File: .github/workflows/prod.yml [hunk 2]
```diff
@@ -1,25 +1,38 @@
-name: Deploy to Production
+name: Deploy to Prod (ECS)
 
 on:
   workflow_dispatch:
+  push:
+    branches: [ps-linux]
```
Comment (Whoops) [inline .github/workflows/prod.yml:6]: here as well

File: mix.exs [hunk 3]
```diff
@@ -10,7 +10,16 @@ defmodule Trike.MixProject do
       deps: deps(),
       test_coverage: [tool: LcovEx, output: "cover"],
       aliases: aliases(),
-      dialyzer: [plt_add_apps: [:mix]]
+      dialyzer: [plt_add_apps: [:mix]],
+      default_release: :trike,
+      releases: [
+        trike: [
+          include_executables_for: [:windows]
```
Comment (Whoops) [inline mix.exs:17]: Do we still need a windows executable?

**Review by Whoops — APPROVED** [review]
Left a couple comments, but it looks good!

File: mix.exs (hunk 3 above)
Comment (nlwstein) [reply mix.exs]: @Whoops Good call, I was able to remove this as well 😄

**Pull request merged**